src/ArticleCard.jsx

```diff
@@ -0,0 +1,24 @@
+export default function ArticleCard({ article }) {
+  const {
+    article_img_url,
+    author,
+    comment_count,
+    created_at,
+    title,
+    topic,
+    votes,
+  } = article;
+
+  return (
+    <>
+      <div class="article-card">
```

Comment:
In React this should be: `className` instead of `class` - if you check your dev tools you'll see an error in the browser console also telling you this.

Reply:
done